fe/fe-core/src/main/java/com/starrocks/catalog/Catalog.java

```diff
@@ -4497,8 +4497,22 @@ public static void getDdlStmt(String dbName, Table table, List<String> createTab
         // 1.1 view
         if (table.getType() == TableType.VIEW) {
             View view = (View) table;
-            sb.append("CREATE VIEW `").append(table.getName()).append("` AS ").append(view.getInlineViewDef());
-            sb.append(";");
+            sb.append("CREATE VIEW `").append(table.getName()).append("` (");
+            List<String> colDef = Lists.newArrayList();
+            for (Column column : table.getBaseSchema()) {
+                StringBuilder colSb = new StringBuilder();
+                colSb.append(column.getName());
+                if (!Strings.isNullOrEmpty(column.getComment())) {
+                    colSb.append(" COMMENT ").append("\"").append(column.getComment()).append("\"");
+                }
+                colDef.add(colSb.toString());
+            }
+            sb.append(Joiner.on(", ").join(colDef));
+            sb.append(")");
+            if (!Strings.isNullOrEmpty(view.getComment())) {
+                sb.append(" COMMENT \"").append(view.getComment()).append("\"");
+            }
+            sb.append(" AS ").append(view.getInlineViewDef()).append(";");
```

Comment:
Is the result in all one line? Is it necessary to add '\n' to be more readable?

Reply:
The logic before this code is not to add line breaks, so keep this logic first, and add it later if the user needs it.